Doc/whatsnew/3.10.rst

```diff
@@ -72,6 +72,59 @@ New Features
 
 .. _whatsnew310-pep563:
 
+Parenthesized context managers
+------------------------------
+
+Using enclosing parentheses for continuation across multiple lines
+in context managers is now supported. This allows formatting a long
+collection of context managers in multiple lines in a similar way
+as it was previously possible with import statements. For instance,
+all these examples are now valid:
+
+.. code-block:: python
+
+    with (CtxManager() as example):
```

Comment:
I can't really see too clearly here, but the first indent to this entire code block should be indented with 3 spaces not 4 (this might result in a visual difference). If I'm mistaken, apologies in advance and please ignore the comment.

Reply:
> The first indent to this entire code block should be indented with 3 spaces not 4

Hummmm.....why is that? My understanding is that all indents should be the same in code blockd

Reply:
![Screenshot from 2021-01-25 23-04-05](https://user-images.githubusercontent.com/11718525/105777089-a8990880-5f61-11eb-9275-ec463151ee18.png)

It renders ok to me